README.md

```diff
@@ -1,6 +1,11 @@
-# Getting Started with Create React App
+# Math-Magician
+
+- This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+- All the linters has been installed correctly
+- WebPack and React has been set-up correctly
+![Mi video](https://user-images.githubusercontent.com/105079888/188669578-0538c98c-735f-4227-b4d8-811410c9dab7.gif)
+
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
```

Comment:
- [ ] Please, consider modifying your README file, and make it more professional by removing the content that came with the `create-react-app` command. I see you have added the title and description for the project, but other sections, like how to set up on the local machine, etc, are missed. You can find all necessary section in the [MICROVERSE template](https://github.com/microverseinc/readme-template/blob/master/README.md), to make the README file looks professional 👍

Reply:
@elmar8287 Hi there Elmar, thank you for your review, Honestly the project is not ready is just the set up, the readme file is Edited almos at the end of the project. Thank you for your review but I believe is something not important to this level.

Reply:
@elmar8287 It is ready Elmar, kindly check and tell me, maybe you would like to make me add the deployment link from now 😄 even if the DashBoard has some errors. Thank you

Reply:
Dear @LuisDavidRodriguez, I am glad that you easily did it. The Readme file is in general requirements and I have to mark it. I think you are familiar with the MICROVERSE template from the first projects. If you read the project requirements carefully, you can see that you have to make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).  Here is also the [Math magicians: project Setup](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_setup.md) requirements
![image](https://user-images.githubusercontent.com/49064106/188730223-1dfbc1ed-a960-4119-a7fe-ed4b7b639f2a.png)

Deployment is optional actually! Happy coding 🥇